src/test/kotlin/com/exactpro/th2/check1/rule/check/TestCheckRuleTask.kt

```diff
@@ -149,8 +148,7 @@ internal class TestCheckRuleTask : AbstractCheckTaskTest() {
         assertEquals(1, eventList.size)
         assertEquals(FAILED, eventList[0].status)
 
-        verify(onTaskFinishedMock).invoke(FAILED)
-        verifyNoMoreInteractions(onTaskFinishedMock)
```

Comment:
Why have you removed `verifyNoMoreInteractions`?

Reply:
I used VerificationMode.only() instead.

Reply:
Okay, got it